src/bitcoin-cli.cpp

```diff
@@ -867,6 +874,84 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+    std::string RESET, GREEN, BLUE, YELLOW, MAGENTA, CYAN;
+
+#ifndef WIN32
+    if (isatty(fileno(stdout))) {
+      // Only print colored text if OS is not WIN32 and stdout is connected to a terminal.
+      RESET = "\x1B[0m";
+      GREEN = "\x1B[32m";
+      BLUE = "\x1B[34m";
+      YELLOW = "\x1B[33m";
+      MAGENTA = "\x1B[35m";
+      CYAN = "\x1B[36m";
+    }
+#endif
+
+    std::string result_string = strprintf("%sChain: %s%s\n", BLUE, result["chain"].getValStr(), RESET);
+    result_string += strprintf("Blocks: %s\n", result["blocks"].getValStr());
+    result_string += strprintf("Headers: %s\n", result["headers"].getValStr());
+    result_string += strprintf("Verification progress: %s\n", result["verificationprogress"].getValStr());
+    result_string += strprintf("Difficulty: %s\n\n", result["difficulty"].getValStr());
+
+    result_string += strprintf(
+        "%sNetwork: in %s, out %s, total %s%s\n",
```

Comment:
nit: Maybe `Connections` is clearer. `Network` can refer to IPv4, Tor, I2P ...

Reply:
`Network` a "header" that is used to represent that the segment contains network-related information from `getnetworkinfo`. The connections information moved up to the same line as `Network` as their values are quite self-explanatory and we will save vertical real estate with that. https://github.com/bitcoin/bitcoin/pull/21832#issuecomment-840096163 for more information.